python_modules/dagster/dagster/_core/execution/resolve_versions.py

```diff
@@ -110,10 +110,10 @@ def resolve_step_versions(
         solid_config = resolved_run_config.solids[solid_name].config
 
         solid_def_version = None
-        if solid_def.version is not None:
-            solid_def_version = solid_def.version
+        if solid_def.version is not None:  # type: ignore  # (should be OpDefinition)
```

Comment:
what is `solid_def` instead? do we just need to update the call above to a different getter?

Reply:
It's `NodeDefinition`, which doesn't have `version`-- will fix this upstack

Reply:
i dont think we should get intermediate Graph nodes here so probably can use whatever getter ensures its returning an `OpDefinition`